docs/src/pages/quasar-cli-vite/convert-to-quasar-cli-with-vite.md

```diff
@@ -64,7 +64,9 @@ A Quasar CLI with Webpack project relies on `/package.json > browserslist` to sp
 More info: [Configuring SSR](/quasar-cli-vite/developing-ssr/configuring-ssr)
 
 ### 6. PWA related
+
 * **VERY important: BEFORE porting your files over, run command `quasar mode add pwa`. Otherwise all the needed packages will not be added, and your build will fail.**
+* If migrating from webpack, please note that the name of the service worker file has changed for vite (to `sw.js`). This can break your update process the first time the new app is loaded. To ensure smooth upgrades from previous webpack builds, make sure the name matches the name of your previous service worker file (usually `service-worker.js`). Look for option `swFilename: 'sw.js'` in `quasar.config.js > pwa`.
```

Comment:
```suggestion
* The default name of the outputted service worker file has changed from `service-worker.js` to `sw.js`. This can break your update process the first time the new app is loaded. So, if your app is in production, to ensure smooth upgrades from the previous Webpack builds, make sure the name matches the name of your previous service worker file. You can set it through [`quasar.config.js > pwa > swFilename`](/quasar-cli-vite/developing-pwa/configuring-pwa#quasar-config-js).
```